inc/block-patterns.php

```diff
@@ -117,7 +119,7 @@ function twentytwentytwo_register_block_patterns() {
 	 *
 	 * @since Twenty Twenty-Two 1.0
 	 *
-	 * @param $block_patterns array List of block patterns by name.
+	 * @param array $block_patterns List of block patterns by name.
```

Comment:
I am not familiar with how phpstan configuration works but the rule that flags this change seems like it may need to be adjusted as it would conflict with core standards for documentation of function params.

Reply:
disregard previous comment, I read the diff wrong on mobile.